share/lib/python/neuron/rxd/node.py

```diff
@@ -350,8 +359,11 @@ def include_flux(self, *args, **kwargs):
                     f = float(args[0])
                     source = f
                     success = True
-                except:
-                    pass
+                except TypeError:
+                    arith = rxdmath._ensure_arithmeticed(args[0])
+                    source = lambda: eval_arith_flux(arith, self.region, self)
```

Comment:
We should check to see if `artih` actually ends up as an `_Arithmeticed` object. If `args[0]` isn't compatible (for example, I think if we passed in a string or a list), then `arith` doesn't become an `_Arithmeticed` so the lambda would fail when called, which probably means that the code would segfault.

Reply:
`rxdmath._ensure_arithmeticed` will always return an `_Arithmeticed` object unless it's argument is a `_Reaction` object. It will set `valid_reaction_term=False` but it will also set that if you call `rxdmath._ensure_arithmeticed` on an object that is already an `_Arithmeticed` object.